template_rpm/distribution.sh

```diff
@@ -119,6 +123,7 @@ function yumInstall() {
     mount --bind "${PACKAGES_DIR}" "${INSTALL_DIR}/tmp/template-builder-repo"
     if [ -e "${INSTALL_DIR}/usr/bin/$DNF" ]; then
         cp "${TEMPLATE_CONTENT_DIR}/template-builder-repo-${DIST_NAME}.repo" "${INSTALL_DIR}/etc/yum.repos.d/"
+        chroot_cmd $DNF config-manager --setopt=deltarpm=False --setopt=zchunk=False --setopt=gpgcheck=1 --save
```

Comment:
As I said, modifications to the template's files should be done via some package. The template build itself you already covered with `DNF_OPTS`.

Reply:
Which package does this belong in?  `core-agent-linux`?